data/json/mapgen/cave.json

```diff
@@ -646,6 +646,94 @@
       "monsters": { " ": { "monster": "GROUP_CAVE", "chance": 2 } }
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "cave_rat_underground" ],
+    "object": {
+      "fill_ter": "t_rock_floor",
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "1...........2...........",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........... ............",
+        "........... ............",
+        "........... ............",
+        "......     >............",
+        "3...........4...........",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "......................  ",
+        "...................... <"
+      ],
+      "terrain": { ".": "t_rock", " ": "t_rock_floor", ">": "t_slope_up", "<": "t_slope_down" },
+      "furniture": {  },
+      "nested": {
+        "1": { "chunks": [ [ "cave_nw_1", 25 ], [ "cave_nw_2", 25 ], [ "cave_nw_3", 25 ], [ "cave_nw_4", 25 ] ] },
```

Comment:
As of #39511 you can just make four nested mapgens with the same `cave_nw` id and specify the weights (though in this case they're all equal weight) to get the same result. Not required, and if you really do want to have explicit mapgen ids for these that's fine too.

Reply:
Thanks for the info.